mono/mini/aot-compiler.c

```diff
@@ -7873,6 +7884,8 @@ mono_aot_parse_options (const char *aot_options, MonoAotOptions *opts)
 			opts->llvm_opts = g_strdup (arg + strlen ("llvmopts="));
 		} else if (str_begins_with (arg, "llvmllc=")){
 			opts->llvm_llc = g_strdup (arg + strlen ("llvmllc="));
+		} else if (str_begins_with (arg, "llvmllcopts=")) {
+			opts->llvm_llc_opts = g_strdup (arg + strlen ("llvmllcopts="));
```

Comment:
I would call this llcmopts. Alternative, we can add a mcpu/march argument similar to the mtuple argument.